packages/constants/index.ts

```diff
@@ -0,0 +1 @@
+export * from './common';
```

Comment:
see example of `odf-console` repo...
index file is only for exporting other files... which means all the files in `constants` directory will be exported from `index` file... and all the constants will be defined in those files which are being exported...

Reply:
now when we import constants anywhere in code... we just have to write `import { constant_name } from 'some_path_here/components/constants' 
that is it... u don't have to mention `index` in this path or any other file name which is being exported from this index file...

Reply:
see example in `odf-console` how we export from any `index` file and how we import from that as well...

Reply:
Done